app/models/take_medication_reminder.rb

```diff
@@ -0,0 +1,5 @@
+class TakeMedicationReminder < ActiveRecord::Base
+  belongs_to :medication
+  validates_inclusion_of :active, in: [true, false]
+  scope :active, -> { where(active: true) }
+end
```

Comment:
I thought about it when I originally was going to use Single Table Inheritance for the two reminders. But I think with them being separate tables, it ends up being a Reminder concern with just 
`validates_inclusion_of :active, in: [true, false] ` and `scope :active, -> { where(active: true) }` which felt like overkill?